static/widgets/site-templates-widget.ts

```diff
@@ -47,7 +47,9 @@ class SiteTemplatesWidget {
     }
     getCurrentTheme() {
         const theme = Settings.getStoredSettings()['theme'];
-        if (theme === 'system') {
+        if (!theme) {
+            return 'default'; // just default to dark if undefined, apparently this can happen
```

Comment:
comment seems wrong, dark is not the default

Reply:
suggest to leave out comment and let git blame do the talking

Reply:
Ope thank you!